rskj-core/src/main/java/co/rsk/peg/BridgeUtils.java

```diff
@@ -159,6 +153,10 @@ public static boolean scriptCorrectlySpendsTx(BtcTransaction tx, int index, Scri
         }
     }
 
+    protected static Coin getAmountSentToWallet(BtcTransaction btcTx, Wallet wallet) {
```

Comment:
Maybe this could receive an address instead of a Wallet, and take care of creating a Wallet watching the giving address. I feel it would be more useful that way